include/libopencm3/stm32/l0/timer.h

```diff
@@ -138,31 +138,53 @@ LGPL License Terms @ref lgpl_license
 #define LPTIM_IER_UPIE			(1 << 5)
 #define LPTIM_IER_DOWNIE		(1 << 6)
 
+/* --- LPTIM_CFGR values ----------------------------------------------------- */
```

Comment:
If you're going to add a "blah values" section, you might as well make it a doxygen group.  We've done that in other places, so there's a group for all the values of each register.  Just this ----- line doesn't really do much.

Reply:
hmm, do you have some good doc example/guideline to follow to keep things consistent ? I aligned with timer_common_all.h :)